packages/react-app/src/components/giveaways/details/MainContent.tsx

```diff
@@ -9,11 +9,9 @@ import { GiveawayContext } from '../../../pages/details';
 import { UserContext } from '../../../routes/AuthRoute';
 import PendingApprovalBanner from '../participation/PendingApprovalBanner';
 
-type GiveawayMainContentProps = {
-  participants: Participant[];
-};
+type GiveawayMainContentProps = {};
 
-const GiveawayMainContent = ({ participants }: GiveawayMainContentProps) => {
+const GiveawayMainContent = () => {
```

Comment:
The `details` component should also be updated since we no longer pass the participants as props

Reply:
We still use participants on MainContent to check for winninChance rendering, does that use get removed as well?